components/molecules/MostActiveContributorsCard/most-active-contributors-card.tsx

```diff
@@ -204,12 +204,10 @@ export default function MostActiveContributorsCard({
 
 function RowTooltip({
   contributor,
-  highlightedStat,
   dataLabels,
   children,
 }: {
   contributor: ContributorStat;
-  highlightedStat: StatKeys;
```

Comment:
There didn't seem to be a reason for this aside from bolding the first item in the legend.